public/.well-known/walletconnect.txt

```diff
@@ -0,0 +1 @@
+f1f5f288-9590-4c27-b0b0-05791879e03d=a8cd12deb0fda53adc4045251b82f4e387d04fdf0660141802280ac6a7881331
```

Comment:
Could you include this file on the .gitignore too? And rotate that verification key as well